graders/r/README.md

```diff
@@ -1,10 +1,31 @@
-## PrairieLearn Autograding R Docker Image
+## rocker-pl
 
+This is a Docker image working as an external grader to be paired with PrairieLearn.  It has been
+used in the [STAT 430](https://stat430.com) (2018-2020) and [STAT 447](https://stat430.com) (since
+2021) courses on _Data Science Programming Methods_ at the University of Illinois at
+Urbana-Champaign. In a previous course iteration, a CentOS7-based container was used via Python. We
+now drive R directly from PrairieLearn evaluating code in this container. However, the container is
+perfectly generic and can be used for general R work as well.
 
-The docker container is from the [versioned `tidyverse` stack](https://github.com/rocker-org/rocker-versioned)
-([3.6.1 source](https://github.com/rocker-org/rocker-versioned/blob/master/tidyverse/Dockerfile))
-created by the [`rocker`](https://www.rocker-project.org/) project. Ontop of the 
-versioned `tidyverse` container, we've included packages additional _R_ packages
-and a specific user, called `ag`, for PrairieLearn's autograde routine.
+It is based on the [Rocker](https://rocker-project.org) container `r-ubuntu` in order to take
+advantage of the prebuilt Ubuntu binaries available for the stable 'long-term support' (LTS)
+releases---both current versions of R itself, as well as current CRAN packages.
 
+### R Packages Installed
 
+See the Dockerfile for full details but we include
+
+- tidyverse and data.table for data manipulation;
+- RUnit, testthat and tinytest for testing;
+- microbenchmark, rbenchmark and bench for timing;
+- shiny, flexdashboard, dygraphs for shiny;
```

Comment:
Can we include some example questions in `exampleCourse/questions/demo/autograder/r`? These would parallel the example questions in the neighboring `python`, `c`, and `java` directories.

It would also be great to include a question called `exampleCourse/questions/demo/workspace/rstudio` that uses both the new `prairielearn/workspace-rstudio` image and grades the resulting code with this autograder.

Reply:
That is planned.  See https://github.com/eddelbuettel/pl-r-demos   -- and I think you I even briefly riffed on Slack that we want to bring this in first, then adjust the question and bring the questions in.

Reply:
With that, any other outstanding issues you would like to see addressed, and that are not covered in my push earlier?

Reply:
Just a decision about https://github.com/PrairieLearn/PrairieLearn/pull/4699#discussion_r701345955 --- can we either put the (unpacked) dev version of `plr` into `graders/r/plr/...` or shift it to `PrairieLearn/plr`? I'm fine either way, just let me know which one you prefer.